src/MetricsUploader/MetricsUploaderWindows.cpp

```diff
@@ -181,7 +181,8 @@ ErrorMessageOr<std::string> GenerateUUID() {
 }
 
 bool MetricsUploaderImpl::SendLogEvent(OrbitLogEvent_LogEventType log_event_type,
-                                       std::chrono::milliseconds event_duration) {
+                                       std::chrono::milliseconds event_duration,
+                                       OrbitLogEvent_StatusCode status_code) {
```

Comment:
Maybe we can use partially filled `OrbitLogEvent` as an argument instead of log_event_type, event_duration and status_code (I guess list will grow...)?

Reply:
I am unsure how I feel about having an API for MetricsUploader, where the user has to provide "half" a proto. I rather have some overloads, like I did now. I don't think this will grow much much further. ptal at my comment in the "main thread"